packages/notifi-dapp-example/src/components/TopicRow.tsx

```diff
@@ -89,6 +93,17 @@ export const TopicRow = <T extends TopicRowCategory>(
     }
   };
 
+  const correctUserInputParamsOrder = (
+    userInputParams: UserInputParam<UiType>[],
+  ) => {
+    if (userInputParams.length > 0 && userInputParams[0].uiType === 'radio') {
+      return userInputParams;
+    } else {
+      const reversedParams = [...userInputParams].reverse();
+      return reversedParams;
+    }
+  };
+
```

Comment:
NOTE: per talk, we might also want add a todo or a ticket so that we do not forget to remove this section

Reply:
https://notifi.atlassian.net/browse/MVP-5238 I have the ticket created, feel free to edit if there are some thing that I missed